### PR TITLE
Add units to measurements and better ontology support for observations

### PR DESCRIPTION
**Pull request by Lalaland**
This adds units and improves the ontology support for the observation table

### CODE REVIEW

File: tools/stanford/flowsheet_cleaner.py [hunk 1]
```diff
@@ -69,9 +84,12 @@ def convert_row(row: Mapping[str, str]) -> Optional[Dict[str, str]]:
         return None
 
 
-def get_new_sources(root: str, child: str) -> Set[str]:
+def get_new_sources(
```
Comment (jason-fries) [inline tools/stanford/flowsheet_cleaner.py:87]: Can we call this something more clear than `get_new_sources` like `map_unknown_concepts`?

File: tools/stanford/flowsheet_cleaner.py [hunk 2]
```diff
@@ -85,13 +103,22 @@ def get_new_sources(root: str, child: str) -> Set[str]:
...
                     new_concepts.add(new_row["unit_source_value"])
     except Exception as e:
         traceback.print_exc()
         raise RuntimeError("Failed " + root + " , " + child, e)
 
-    return new_concepts
+    return new_concepts, new_relationships
 
 
 def correct_rows(
```
Comment (jason-fries) [inline tools/stanford/flowsheet_cleaner.py:124]: In documentation, anytime a claim is made, we should minimally document why that claim is true.  For this function, for example, why are some mappings incorrect? What is a description of the fix in plain language for? etc

File: tools/stanford/flowsheet_cleaner.py [hunk 3]
```diff
@@ -215,6 +251,44 @@ def correct_rows(
...
+
+        with io.TextIOWrapper(
+            zstandard.ZstdCompressor(1).stream_writer(
+                open(destination_rel_path, "wb")
+            )
+        ) as o2:
+            rel_writer = csv.DictWriter(
+                o2,
+                fieldnames=[
+                    "concept_id_1",
```
Comment (jason-fries) [inline tools/stanford/flowsheet_cleaner.py:263]: Why are we using `concept_id_1` and `concept_id_2` -- are these standard field names from BigQuery/OMOP? `child_concept_id` and `parent_concept_id` are much more descriptive.

**Review by jason-fries — APPROVED** [review]
Minor comments. Looks fine overall.

File: tools/stanford/flowsheet_cleaner.py (hunk 3 above)
Comment (Lalaland) [reply tools/stanford/flowsheet_cleaner.py:263]: This is per the omop schema.

**Pull request merged**